docs/model_configuration.md

```diff
@@ -501,6 +501,60 @@ the device kind of the instance, for instance, KIND_CPU is "cpu", KIND_MODEL is
   ]
 ```
 
+### Rate Limiter Config
+
+Instance group optionally specifies [rate limiter](rate_limiter.md)
+config which controls how the rate limiter operates on the instances
+in the group. The rate limiter configuration includes the following
+specifications:
+
+#### Resources
+
+The set of resources required to execute a model instance. Essentially,
+a resource is just mapping of a name to its corresponding count.
+By default, the resource counts are local to the device running the
+instance. However, a resource, with `global` flag set to `True`, will
+be shared across the system. Loaded models can not specify a resource
+with the same name both as global and non-global.
+
+#### Priority
+
+Priority specifies the priority of each instance in the group. For an
+instance, priority serves as a weighting value to be used for
```

Comment:
Isn't the priority set for all instances in that instance group? From the protobuf it appears the priority is paired to an instance group and not a specific instance

Reply:
Yes.. I see why this might become confusing. I will refine the language.